crates/topos-tce-api/src/runtime/mod.rs

```diff
@@ -137,6 +137,10 @@ impl Runtime {
                 Some(command) = self.runtime_command_receiver.recv() => {
                     self.handle_runtime_command(command).await;
                 }
+
+                Some(result) = self.running_sync_tasks.next() => {
+                    info!("The task has been completed with result: {:?}", result);
```

Comment:
It's not very clear to me what "the task" actually is, and why – as a user – I'd want to see the result logged at the `info` level. It would be better imo to let tasks log their final result (if any and if relevant to users) rather than do it here?